src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -8,7 +8,25 @@ public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws InvalidDataException {
+        if (user.getAge() != null && user.getLogin() != null && user.getPassword() != null) {
```

Comment:
The condition `user.getAge() != null && user.getLogin() != null && user.getPassword() != null` should be expanded to include checks for null values for each field separately, with appropriate error messages. Throwing a `NullPointerException` at the end is not a good practice.